.github/PULL_REQUEST_TEMPLATE/PRIMARY.md

```diff
@@ -0,0 +1,34 @@
+**Describe the PR 🚀**
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+**Type of change**
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+**How Has This Been Tested?**
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Manual Testing
+- [ ] Automated Testing
+
+**Test Configuration**:
+
+* SDK:
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
```

Comment:
Should be
- [ ] I have commented on my code, particularly in hard-to-understand areas

Reply:
fixed